mypy/messages.py

```diff
@@ -1005,6 +1004,11 @@ def disallowed_any_type(self, typ: Type, context: Context) -> None:
             message = 'Expression type contains "Any" (has type {})'.format(self.format(typ))
         self.fail(message, context)
 
+    def incorrectly_returning_any(self, typ: Type, context: Context) -> None:
+        message = 'Returning Any from function with declared return type {}'.format(
```

Comment:
This error might sound awkward in situations like:
```
Returning Any from function with declared return type <nothing>
Returning Any from function with declared return type overloaded function
Returning Any from function with declared return type Module
```
I think it will sound better if you say something like
```
Returning Any from function declared to return <nothing>
Returning Any from function declared to return overloaded function
Returning Any from function declared to return Module
```